tools/run-wpt.mjs

```diff
@@ -77,6 +77,16 @@ const VERBOSE = process.env.VERBOSE || 'false';
 // The path to the WPT report file.
 const WPT_REPORT = process.env.WPT_REPORT || 'wptreport.json';
 
+const LOG_DIR = process.env.LOG_DIR || 'logs';
```

Comment:
nit: move these two new definitions upwards so that the variables keep their alphabetical order

Reply:
done